src/content/core/namespaces.mdx

```diff
@@ -1,53 +1,58 @@
 ---
-title: Secure and isolated namespaces
-description: Create and manage secure and isolated Kubernetes namespaces for you and your team
+title: Secure and isolated Namespaces
+description: Create and manage secure and isolated Kubernetes Namespaces for you and your team
```

Comment:
I think the "secure and isolated" phrase is unnecessary now that we don't do multitenancy

Reply:
Makes sense! This is a good opportunity for us to introduce Dev Environments in the description as well.
Addressed this and the rest of your feedback in the latest commit